<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryWindow</class>
 <widget class="QWidget" name="QueryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qt200.qrc">
    <normaloff>:/resource/image/Title.png</normaloff>:/resource/image/Title.png</iconset>
  </property>
  <widget class="QLabel" name="lbNumber">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNumble">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>14</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbName">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>14</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dteStartDate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbCheckTime">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dteEndDate">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQueryData">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Query</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrint">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>60</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twPatientInfo">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>109</y>
     <width>766</width>
     <height>246</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrePage">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pre Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNextPage">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModify">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPageTag">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPostback">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PostBack</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qt200.qrc"/>
 </resources>
 <connections/>
</ui>
